Code/Tools/AssetProcessor/native/utilities/PlatformConfiguration.cpp

```diff
@@ -1125,6 +1136,16 @@ namespace AssetProcessor
 
         MetaDataTypesVisitor visitor;
         settingsRegistry->Visit(visitor, AZ::SettingsRegistryInterface::FixedValueString(AssetProcessorSettingsKey) + "/MetaDataTypes");
+
+        AZStd::vector<AZStd::string> supportedFileExtensions;
+        AssetImporterPathsVisitor assetImporterVisitor{ settingsRegistry, supportedFileExtensions };
+        settingsRegistry->Visit(assetImporterVisitor, AZ::SettingsRegistryInterface::FixedValueString(AssetImporterSettingsKey) + "/SupportedFileTypeExtensions");
```

Comment:
Is there a way you can either share code for looking this up with the SceneImportRequestHandler, just use the SceneImportRequestHandler here, or at least make this string a const you pull from one location for both.

If someone goes and changes "SupportedFileTypeExtensions" in SceneImporteriRequestHandler.cpp, this code will stop working correctly.